templates/registration/logged_out.html

```diff
@@ -0,0 +1,6 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <div style="margin: 25px">You have been logged out!
```

Comment:
Avoid using inline styles. It's better to move this CSS to a separate file and use a class to apply it. This makes your code cleaner and easier to maintain.